src/components/grid/grid-container/index.d.ts

```diff
@@ -23,6 +26,26 @@ export interface GridContainerProps {
    * Overrides default vertical paddings
    */
   py?: string | number;
+  /** Any valid CSS value or a number to be multiplied by base spacing unit (8). Overrides default margin */
+  m?: string | number;
+  /** Any valid CSS value or a number to be multiplied by base spacing unit (8). Overrides default margin-left */
+  ml?: string | number;
+  /** Any valid CSS value or a number to be multiplied by base spacing unit (8). Overrides default margin-right */
+  mr?: string | number;
+  /** Any valid CSS value or a number to be multiplied by base spacing unit (8). Overrides default margin-top */
+  mt?: string | number;
+  /** Any valid CSS value or a number to be multiplied by base spacing unit (8). Overrides default margin-bottom */
+  mb?: string | number;
+  /**
+   * Any valid CSS value or a number to be multiplied by base spacing unit (8).
+   * Overrides default horizontal margins
+   */
+  mx?: string | number;
+  /**
+   * Any valid CSS value or a number to be multiplied by base spacing unit (8).
+   * Overrides default vertical margins
+   */
+  my?: string | number;
```

Comment:
We can use SpacingProps for margin and padding connected props. 
`import { SpacingProps } from "../../../utils/helpers/options-helper";`